fem/tmop_tools.cpp

```diff
@@ -155,7 +155,7 @@ void AdvectorCG::ComputeAtNewPositionScalar(const Vector &new_nodes,
 
    double t = 0.0;
    bool last_step = false;
-   for (int ti = 1; !last_step; ti++)
+   for ( ; !last_step; )
```

Comment:
```suggestion
   while (!last_step)
```

Equivalent, but more idiomatic?

Reply:
I'm fine with it either way.